packages/connectivity/src/scp-cf/destination/destination-from-vcap.ts

```diff
@@ -24,19 +30,45 @@ const logger = createLogger({
  */
 export async function destinationForServiceBinding(
   serviceInstanceName: string,
-  options: DestinationForServiceBindingsOptions = {}
+  options: DestinationForServiceBindingsOptions & {
+    jwt?: JwtPayload;
+    useCache?: boolean;
+  } = {}
```

Comment:
[nth] Define `{ jwt?: JwtPayload; useCache?: boolean; }` as a type outside of the function params so it's easier to read the function signature.